test/helpers/IntegrationHelper.js

```diff
@@ -387,6 +387,39 @@ var customMatchers = {
                 return result;
             }
         };
+    },
+    toAllBeInBottomOfDeck: function () {
+        return {
+            compare: function (cards, player, numCards) {
+                var result = {};
+                const deck = player.deck;
+                const L = deck.length;
+                result.pass = L >= numCards;
+                if (result.pass) {
+                    for (let card of cards) {
+                        result.pass = card.location === 'deck';
+                        if (!result.pass) {
+                            result.message = `Expected ${card.title} to be in the deck.`;
```

Comment:
Should we be appending these strings together to have a message with all of the missing cards? Ditto for line 413.